mining.go

```diff
@@ -20,13 +20,9 @@ import (
 )
 
 const (
-	// generatedBlockVersion is the version of the block being generated.
-	// It is defined as a constant here rather than using the
-	// wire.BlockVersion constant since a change in the block version
-	// will require changes to the generated block.  Using the wire constant
-	// for generated block version could allow creation of invalid blocks
-	// for the updated version.
-	generatedBlockVersion = 4
+	// minHighPriority is the minimum priority value that allows a
+	// transaction to be considered high priority.
+	minHighPriority = btcutil.SatoshiPerBitcoin * 144.0 / 250
```

Comment:
Nice catch.  It was added before the `mempool` package existed and I missed removing it when rebasing over it.